spec/controllers/githook_controller_spec.rb

```diff
@@ -3,16 +3,16 @@
 RSpec.describe GithookController, type: :controller do
   describe "githook", :type => :request do
     it "requires authentication" do
-      get "githook/master"
+      get "/githook/master"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.